dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/registry/WorkerRegistry.java

```diff
@@ -152,6 +152,7 @@ public void unRegistry() {
             // trim and lower case is need
             workerZkPathBuilder.append(workGroup.trim().toLowerCase()).append(SLASH);
             workerZkPathBuilder.append(address);
+            workerZkPathBuilder.append(weight).append(SLASH);
```

Comment:
Hi,

If user choose to use `RoundRobinSelector`, there will be no need to add the `weight` to zk heartbeat info. Maybe we can design this feature better.

Reply:
Hi, in my opinion, every load choice is related to weight. Can you talk about why it is not needed when using the RoundRobin algorithm?